src/models/callercalleemodel.cpp

```diff
@@ -42,30 +43,45 @@ QVariant CallerCalleeModel::headerCell(Columns column, int role)
             return Qt::DescendingOrder;
         }
     }
-    if (role != Qt::DisplayRole) {
+    if (role != Qt::DisplayRole && role != Qt::ToolTipRole) {
         return {};
     }
 
-    switch (column) {
-        case Symbol:
-            return tr("Symbol");
-        case Binary:
-            return tr("Binary");
-        case SelfCost:
-            return tr("Self Cost");
-        case InclusiveCost:
-            return tr("Inclusive Cost");
-        case Callers:
-            return tr("Callers");
-        case Callees:
-            return tr("Callees");
+    if (role == Qt::DisplayRole) {
+        switch (column) {
+            case Symbol:
+                return tr("Symbol");
+            case Binary:
+                return tr("Binary");
+            case SelfCost:
+                return tr("Self Cost");
+            case InclusiveCost:
+                return tr("Inclusive Cost");
+            case Callers:
+                return tr("Callers");
+            case Callees:
+                return tr("Callees");
+        }
+    } else if (role == Qt::ToolTipRole) {
+        switch (column) {
+            case Symbol:
+                return tr("The symbol's function name. May be empty when debug information is missing.");
```

Comment:
you are the native speaker, but isn't it "The symbols function name", i.e. without the apostroph?